src/bootstrap/mk/Makefile.in

```diff
@@ -126,6 +126,8 @@ ci-mingw-x:
 	$(Q)$(CFG_SRC_DIR)/x test --stage 2 $(TEST_SET1)
 ci-mingw-bootstrap:
 	$(Q)$(BOOTSTRAP) test --stage 2 $(TEST_SET2)
+ci-mingw-bootstrap-2:
+	$(Q)$(BOOTSTRAP) test --stage 2 $(TEST_SET1)
```

Comment:
Is there a specific reason

- `ci-mingw-bootstrap` does `TEST_SET2`
- but `ci-mingw-bootstrap-` does `TEST_SET1`

`ci-mingw-x` above does `TEST_SET1` (though there's also `ci-mingw-x-{1,2}` where those test `TEST_SET2` 🤔

Reply:
no specific reason, do you have ideas on how to improve naming?

Reply:
Oh...